src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,82 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (value: number) => void;
+};
+
+export const Pagination: FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const amountOfPages = getNumbers(1, Math.ceil(total / perPage));
+
+  const handlePageChange = (
+    event: React.MouseEvent<HTMLLIElement, MouseEvent>,
+  ) => {
+    const element = event.target as HTMLElement;
+    const page = element.textContent;
+
+    onPageChange(Number(page));
+  };
+
+  const handleArrowPageChange = (direction: string) => {
+    if (direction === 'next' && currentPage !== amountOfPages.length) {
+      onPageChange(currentPage + 1);
+    }
+
+    if (direction === 'prev' && currentPage !== 1) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={cn('page-item', { disabled: currentPage === 1 })}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={() => handleArrowPageChange('prev')}
+        >
+          «
+        </a>
+      </li>
```

Comment:
Using an anchor tag with `href="#prev"` will change the URL fragment when clicked. This may lead to unexpected behavior in a SPA. It's better to use a button element for actions that don't navigate to a different URL.